developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_29.rst

```diff
@@ -26,7 +26,7 @@ Front-end changes
 Changed APIs
 ^^^^^^^^^^^^
 
-* `IAppConfig` has been fully reworked, most of previous methods are now deprecated. The new version of the API implements multiple settings for app config values to define their laziness and sensitivity.
+* `IAppConfig` has been fully reworked, most of previous methods are now deprecated. The new version of the API implements multiple settings for app config values to define their laziness and sensitivity. See also the :ref:`AppConfig<app config>` section.
```

Comment:
```suggestion
* `IAppConfig` has been fully reworked, most of previous methods are now deprecated. The new version of the API implements multiple settings for app config values to define their laziness and sensitivity. See also the :ref:`app config` section.
```

Fine either way, but AFAIK I think the automatic titling will work here (so `AppConfig` should be pulled in from the section heading on its own).